test/item_test.rb

```diff
@@ -20,6 +20,6 @@ def test_it_stores_a_name
 
   def test_it_stores_a_unit_price
     item = Item.new({:unit_price => 32301})
-    assert_equal BigDecimal.new(32301), item.unit_price
+    assert_equal BigDecimal.new("32301")/100, item.unit_price
```

Comment:
I don't think we can do this here. The user wouldn't input `BigDecimal.new("32301")/100`, I think they would input `BigDecimal.new("32301")`, as is in the spec.